src/libs/Database.ts

```diff
@@ -0,0 +1,25 @@
+import * as mongoose from "mongoose";
+class Database {
+  static open = (mongoURL: string) => {
+    return new Promise((resolve, reject) => {
+      mongoose
+        .connect(mongoURL,
+          { useNewUrlParser: true, useUnifiedTopology: true },
+          err => {
+            if (err) {
+              console.log("Error in mongoDB connection");
+              reject(err);
+            }
+            resolve();
+            console.log("DB is connected successfully");
+          }
+        )
+        .catch(error => console.log(error));
```

Comment:
Either use callback or promise. Also, mongoose doesn't support promise for connection.